Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2815,6 +2815,19 @@ - (void)_addAppHighlightInNewGroup:(BOOL)newGroup originatedInApp:(BOOL)originat
 }
 
 - (NSUUID *)_enableTextIndicatorStylingAfterElementWithID:(NSString *)elementID
+{
+    return [self _enableSourceTextAnimationAfterElementWithID:elementID];
+}
+- (NSUUID *)_enableTextIndicatorStylingForElementWithID:(NSString *)elementID
```

Comment:
is this name (`_enableTextIndicatorStylingForElementWithID`) still accurate?

Reply:
I have these here so that Safari doesn't break, but will not that they all just call the new names :)